doc/modules/ROOT/pages/index.adoc

```diff
@@ -17,61 +17,61 @@ include::{templates-url}/microservices/what-you-will-learn.adoc[]
 
 - ~15 minutes
 - Apache Maven 3.2+
-- Jdk 1.8+
+- JDK 1.8+
 
 == The Micronaut Application Structure
 
-The application in the initial directory is a basic Micronaut app having 3 endpoints:
+The application in the initial directory is a basic Micronaut app having three endpoints:
 
- -   / is the homepage returning “Homepage” string only
- -   /put is the mapping where key and value is saved to a local map through @CachePut annotation.
- -   /get is the mapping where the values in the local map can be obtained by keys through @Cacheable annotation.
+-   `/` is the homepage returning “Homepage” string only
+-   `/put` is the mapping where key and value are saved to a local map through `@CachePut` annotation.
+-   `/get` is the mapping where the values in the local map can be obtained by keys through `@Cacheable` annotation.
 
 
 == Using Hazelcast in the Application
 
-To configure caching with Hazelcast, firstly we will add some dependencies to our pom.xml file:
+To configure caching with Hazelcast, we'll start by adding some dependencies to our `pom.xml` file:
 [source,xml]
 ----
-        <dependency>
-            <groupId>io.micronaut.cache</groupId>
-            <artifactId>micronaut-cache-hazelcast</artifactId>
-            <version>${micronaut-cache-hazelcast.version}</version>
-        </dependency>
-        <dependency>
-            <groupId>com.hazelcast</groupId>
-            <artifactId>hazelcast</artifactId>
-            <version>${hazelcast.version}</version>
-        </dependency>
+<dependency>
+    <groupId>io.micronaut.cache</groupId>
+    <artifactId>micronaut-cache-hazelcast</artifactId>
+    <version>${micronaut-cache-hazelcast.version}</version>
+</dependency>
+<dependency>
+    <groupId>com.hazelcast</groupId>
+    <artifactId>hazelcast</artifactId>
+    <version>${hazelcast.version}</version>
+</dependency>
 ----
-The first dependency is for Micronaut Cache for Hazelcast, the second one is for Hazelcast IMDG itself,
+The first dependency is Micronaut Cache for Hazelcast, the second one is Hazelcast IMDG itself.
 
 Now, we just need to add a configuration bean to enable Hazelcast:
 [source,java]
 ----
-@Singleton
-public class HazelcastAdditionalSettings
-        implements BeanCreatedEventListener<HazelcastMemberConfiguration> {
+@Factory
+public class HazelcastConfiguration {
 
-    public HazelcastMemberConfiguration onCreated(BeanCreatedEvent<HazelcastMemberConfiguration> event) {
-        HazelcastMemberConfiguration configuration = event.getBean();
-        configuration.setClusterName("micronaut-cluster");
+    @Bean
+    public Config hazelcastConfig() {
+        Config configuration = new Config()
+          .setClusterName("micronaut-clusterx");
```

Comment:
It might be better to remove the `x` in the end :) I'd put it to differentiate it from the old one.